scripts/styleguide.config.js

```diff
@@ -98,6 +98,7 @@ const allSections = [
             '../src/components/text-area/TextArea.js',
             '../src/components/text-input/TextInput.js',
             '../src/components/text-input-with-copy-button/TextInputWithCopyButton.js',
+            '../src/components/theme/index.js',
```

Comment:
I believe the new standard is to create Storybook stories rather than Styleguidist examples.

Reply:
I'll be looking to create the story for this component in the next iteration where I can get a color knob in place as well.